### PR TITLE
Update Lis cmdlets

### PR DESCRIPTION
**Pull request by JensTrier**
Adding examples and format information for latitude and longitude. Various other smaller updates.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@get-itips @dariomws @tseward

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/New-CsOnlineLisCivicAddress.md [hunk 1]
```diff
@@ -262,7 +255,7 @@ Accept wildcard characters: False
 ```
 
 ### -Longitude
-Specifies the angular distance of a place east or west of the meridian at Greenwich, England.
+Specifies the angular distance of a place east or west of the meridian at Greenwich, England using the decimal degrees format.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsOnlineLisCivicAddress.md:258]:
```suggestion
Specifies the angular distance of a place east or west of the meridian at Greenwich, England, using the decimal degrees format.
```

File: skype/skype-ps/skype/New-CsOnlineLisLocation.md [hunk 2]
```diff
@@ -296,7 +296,7 @@ Accept wildcard characters: False
 ```
 
 ### -Longitude
-The longitude of the civic address.
+Specifies the angular distance of a place east or west of the meridian at Greenwich, England using the decimal degrees format.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsOnlineLisLocation.md:299]:
```suggestion
Specifies the angular distance of a place east or west of the meridian at Greenwich, England, using the decimal degrees format.
```

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the minor suggested changes. Many thanks.

@JohanFreelancer9 thanks and done!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tseward Gentle reminder

**Comment by yogkumgit** [comment]
@tiburd Pleas merge this PR. Thanks

**Pull request merged**